src/fmt/writer/termcolor/extern_impl.rs

```diff
@@ -103,30 +103,26 @@ impl BufferWriter {
     }
 
     pub(in crate::fmt::writer) fn pipe(
-        is_test: bool,
+        _is_test: bool,
```

Comment:
If this parameter is no longer used, it should be removed.

Reply:
OK done.